spring-boot-project/spring-boot-docs/src/docs/antora/modules/reference/pages/actuator/loggers.adoc

```diff
@@ -31,3 +31,16 @@ To configure a given logger, `POST` a partial entity to the resource's URI, as t
 ----
 
 TIP: To "`reset`" the specific level of the logger (and use the default configuration instead), you can pass a value of `null` as the `configuredLevel`.
+
+== OpenTelemetry
+By default, the OpenTelemetry SDK logs are not configured. You can provide the location of the OpenTelemetry logs endpoint to configure it:
+
+[source,yaml]
```

Comment:
Rather than `[source,yaml]` this should be `[configprops,yaml]`. That will give us validation and a `.properties` version automatically.